core/src/main/java/com/sekwah/advancedportals/core/tags/activation/NameTag.java

```diff
@@ -6,6 +6,7 @@
 import com.sekwah.advancedportals.core.warphandler.ActivationData;
 import com.sekwah.advancedportals.core.warphandler.Tag;
 
+import javax.annotation.Nullable;
 import java.util.List;
```

Comment:
There was some super old issue with nullable which I just avoided but I believe it was the Jetbrains nullable xD I need to start properly using it.